book_maker/loader/epub_loader.py

```diff
@@ -15,6 +17,68 @@
 from .base_loader import BaseBookLoader
 
 
+# ref: https://platform.openai.com/docs/guides/chat/introduction
+def num_tokens_from_text(text, model="gpt-3.5-turbo-0301"):
```

Comment:
Have you checked if this result equals  `completion[usage][total_tokens"]`?

Reply:
`num_tokens_from_text` returns the text part of the input, and `completion[usage][total_tokens"]` is input+output+system_msg+... They should be different

Reply:
Got it.Should it equal completion`["usage"[promt_token"]`?

Reply:
got it.